README.md

```diff
@@ -1,4 +1,33 @@
-# Azure API Manager Blue Green Deploy Function App
+# Azure Search Index Refresh Function App
+
+## Overview
+
+The purpose of this function app is to allow Azure Search indexes to be recreated without any down time. This is achieved by having a fixed endpoint for the search index managed as an API in Azure API Manager. Behind this there will exist an index in Azure Search which conforms to a naming convention captured by the following regular expression:
+
+``` RegEx
+(?<Name>.*)-(?<Version>[0-9]+-[0-9]+|)-(?<Deployment>a|b)-(?<Environment>dev|int|stag|prod)
+```
+
+### Example
+
+API Manager API
+
+* name: service-search-organisations
+* external URL: https://api.nhs.uk/service-search/search
+
+Before the function app has run:
+
+* serviceUrl: https://nhsuksearchprodne.search.windows.net/indexes/organisationlookup-a-dev/docs/
```

Comment:
I'd replace `nhsuksearchprodne` with something a bit more generic e.g. `<search-service-name>` and possibly update the name of the index too.